src/main.ts

```diff
@@ -238,6 +251,41 @@ const run = async (): Promise<void> => {
       objectDebug('uploadResult', uploadResult)
     }
 
+    if (options.dashboard) {
+      let dashboardResponse
+      const dashboardPayload = {
+        channel: options.channel,
+        context,
+        githubRunId,
+        isUpdating: Boolean(messageTimestamp),
+        jobs,
+        slack: {
+          ...renderResult,
+          channel: options.channel,
+          ts: messageTimestamp,
+        },
+      }
+
+      log.info(`dashboard payload: ${dashboardPayload}`)
+
+      try {
+        dashboardResponse = await fetch(options.dashboard, {
+          body: JSON.stringify(dashboardPayload),
+          headers: {
+            'Content-type': 'application/json',
+          },
+          method: 'POST',
+        })
+      } catch (error: unknown) {
+        log.error(error)
+      }
+
+      if (dashboardResponse?.ok) {
+        const dashboardResponseBody = await dashboardResponse.json()
```

Comment:
Missing error handling. Doesn't check content type before parsing JSON either.

Reply:
Isn't this almost exactly the same logic used to send the request to Slack?

Reply:
Good call! We should check if content-type doesn't include `application/json` and catch the `.json()` parsing, throwing a specific error in both cases, like `MalformedReponse`. This might be a reason that from time to time we still get the action to break the workflow because random slack API errors.

In case of such error, not sure if just ignore, or retry. We don't have retry for `SlackCommunicationError`s either. So we either ignore both, or add a retry to both cases.